klasses/api.py

```diff
@@ -3,13 +3,32 @@
 """
 from decimal import Decimal
 
+from applications.constants import AppStates
 from ecommerce.models import Line
 from ecommerce.serializers import LineSerializer
-from klasses.bootcamp_admissions_client import BootcampAdmissionClient
 from klasses.models import BootcampRun
 from klasses.serializers import InstallmentSerializer
 
 
+def payable_bootcamp_run_keys(user):
+    """
+    Fetches a list of bootcamps for which the user has applied
+
+    Args:
+        user (User): a user
+
+    Returns:
+        list of BootcampRun: bootcamp runs the user has applied for
+
+    """
+    return (
+        BootcampRun.objects.prefetch_related("applications")
+            .filter(applications__user=user)
+            .filter(applications__state=AppStates.AWAITING_PAYMENT.value)
+            .values_list("run_key", flat=True)
```

Comment:
@gsidebo I wrote this as a quick replacement for the old `BootcampAdmissionClient. payable_bootcamp_run_keys` function, just to keep the PR development moving.  Do these criteria make sense at all or should I just wait until #447 is complete?

Reply:
@mbertrand You can combine those filters into one, but that's just a nit. This is a perfectly good replacement while you're hacking on this. You might already be going for this, but just to make sure, my impression was that the `BootcampAdmissionClient` class would be removed entirely as part of this PR. If that class needs to stay for any reason, let me know.

One other comment: please put https://github.com/mitodl/bootcamp-ecommerce/issues/478 in the relevant tickets section as well. That issue has some more detail and some rough acceptance criteria I wrote up (sorry, there are lots of duplicates for this!)

Reply:
Another thing: https://github.com/mitodl/bootcamp-ecommerce/issues/447 is listed as a dependency, but looking at it now, I think it would be fine to merge this without that endpoint. We're getting rid of SMApply and Jobma as part of this PR anyway, which removes the existing endpoint(s) that return payable bootcamps. In production, users won't be using our new application process for some time, so I don't think it's necessary to have a ready-made replacement for those endpoints. I'll make a note of it on the issue